wagtail/documents/tests/test_admin_views.py

```diff
@@ -371,6 +372,33 @@ def test_tag_filtering_with_search_term(self):
         response = self.get({"tag": "one", "q": "test"})
         self.assertEqual(response.context["page_obj"].paginator.count, 2)
 
+    def test_search_and_order_by_created_at(self):
+        # Create Documents
+        doc1 = models.Document.objects.create(title="recent good Document")
+        doc1.created_at = local_datetime(2024, 1, 1)
+        doc1.save()
+
+        doc2 = models.Document.objects.create(title="latest good Document")
+        doc2.created_at = local_datetime(2025, 1, 1)
+        doc2.save()
+
+        old_doc = models.Document.objects.create(title="Oldest good document")
+        old_doc.created_at = local_datetime(2023, 1, 1)
+        old_doc.save()
+
+        response = self.get({"q": "good", "ordering": "created_at"})
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.context["query_string"], "good")
+
+        # Check that the documents are ordered by created_at (oldest first)
+        documents = list(response.context["page_obj"].object_list)
+        self.assertEqual(documents, [old_doc, doc1, doc2])
```

Comment:
To ensure the search works, we should create a document that doesn't have the word "good" in the title and make sure it doesn't show up in the results.